### PR TITLE
Add In-game Heads-Up Display

### PR DESCRIPTION
**Pull request by francoisbaudouin**

### CODE REVIEW

File: locales/locale_fr.po [hunk 1]
```diff
@@ -55,6 +55,18 @@ msgstr "Jouer"
 msgid "Quit"
 msgstr "Quitter"
 
+msgid "number Bomb"
+msgstr "nombre de Bomb"
```
Comment (ZQUEMA) [inline locales/locale_fr.po:59]: Bad traduction

File: locales/locale_fr.po [hunk 2]
```diff
@@ -55,6 +55,18 @@ msgstr "Jouer"
...
+
+msgid "Speed"
+msgstr "Vitesse"
+
+msgid "Power Bomb"
+msgstr "Puissance Bomb"
```
Comment (ZQUEMA) [inline locales/locale_fr.po:65]: Bad traduction

File: locales/locale_en.po [hunk 3]
```diff
@@ -55,6 +55,18 @@ msgstr ""
 msgid "Quit"
 msgstr ""
 
+msgid "Bomb number"
```
Comment (ZQUEMA) [inline locales/locale_en.po:58]: Bombs

File: src/game/components/Hud.cpp [hunk 4]
```diff
@@ -0,0 +1,10 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Hud
+*/
+
+#include "Hud.hpp"
+
```
Comment (ZQUEMA) [inline src/game/components/Hud.cpp:9]: Why does this file exist?

**Review by ZQUEMA — CHANGES_REQUESTED** [review]
Incorrect scaling on the screen

File: src/game/components/Hud.hpp [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    struct Hud : public ecs::Component {
+        ///
+        size_t nbBomb;
+        size_t rangeBomb;
+        float speed;
+        bool inverted;
+        bool slowness;
+        Identity::Id idHud;
+
+        // Hud(Player owner, Identity id)
```
Comment (AndreasLrx) [inline src/game/components/Hud.hpp:29]: Trailing comments

File: src/game/systems/DrawHud.hpp [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        void run(ecs::SystemData data) override final;
+
+      private:
+        void drawBorder(float x, float y);
+        void drawDataHud(float x, float y, std::string text, raylib::core::Color color);
+
+        raylib::core::Color whatColor(components::Player::Inventory inv, int id, float x, float y);
+    };
+} // namespace game::systems
+#endif /*!DRAW_HUD_HPP_*/
```
Comment (AndreasLrx) [inline src/game/systems/DrawHud.hpp:34]: Missing newline

File: src/localization/Resources.hpp [hunk 7]
```diff
@@ -110,7 +110,23 @@ namespace localization
             constexpr ResourceString rsNotConnected("Not connected");
             constexpr ResourceString rsConnected("Connected");
         } // namespace menu
-    }     // namespace resources
+        namespace hud
+        {
+            constexpr ResourceString rsNumberbomb("Bombs");
+            constexpr ResourceString rsSpeed("Speed");
+            constexpr ResourceString rsPowerBomb("Power range");
+            constexpr ResourceString rsNoclip("NoClip");
```
Comment (AndreasLrx) [inline src/localization/Resources.hpp:118]: Rename to Wall pass

**Review by AndreasLrx — CHANGES_REQUESTED** [review]

**Review by AndreasLrx — APPROVED** [review]

**Comment by AndreasLrx** [comment]
Duplicate #298

**Pull request closed without merging**